app/templates/admin/createEvent.html

```diff
@@ -44,7 +44,7 @@ <h3>CELTS-Sponsored Event</h3>
       {% endif %}
     <h1 id="title">{{page_title}}</h1>
     {% if isPastEvent %}
-      <div class="alert alert-danger" role="alert">This event is in the past.</div>
+      <h3>This is a past event!</h3>
```

Comment:
No, bad. This is also not a change that is reflected on trackVolunteers or viewEvent.